Automattic/vip-go-mu-plugins
# Update http-concat submodule to 908cf148

Add `js_concat_script_attributes` filter for additional script attributes.

Example usage:
```
add_filter( 'js_concat_script_attributes', function( $args, $href, $js_array, $jsconcat ) {

    return stristr( $href, '_static' ) !== false ? [ 'async', 'defer' ] : [];

}, 10, 4 );
```

## Description
See https://github.com/Automattic/nginx-http-concat/pull/64

## Changelog Description

### Plugin updated: nginx-http-concat
Add `js_concat_script_attributes` filter to allow for adding script attributes to concatenated scripts. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR
2. Use filter like in example above
3. Check HTML contents to see the attributes being added
4. Have some ice cream
